tests/Protocol/ParserTest.php

```diff
@@ -28,12 +28,12 @@ public function testConvertTcpDumpToBinary($expected, $data)
 
     public function provideConvertTcpDumpToBinary()
     {
-        return array(
-            array(chr(0x72).chr(0x62), "72 62"),
-            array(chr(0x72).chr(0x62).chr(0x01).chr(0x00), "72 62 01 00"),
-            array(chr(0x72).chr(0x62).chr(0x01).chr(0x00).chr(0x00).chr(0x01), "72 62 01 00 00 01"),
-            array(chr(0x01).chr(0x00).chr(0x01), "01 00 01"),
-        );
+        return [
+            [chr(0x72).chr(0x62), "72 62"],
+            [chr(0x72).chr(0x62).chr(0x01).chr(0x00), "72 62 01 00"],
+            [chr(0x72).chr(0x62).chr(0x01).chr(0x00).chr(0x00).chr(0x01), "72 62 01 00 00 01"],
+            [chr(0x01).chr(0x00).chr(0x01), "01 00 01"],
+        ];
```

Comment:
Using `yield` makes the code IMHO cleaner to read, especially with bigger sets. Could also added a label with additional clearance with `yield 'label' => [];` instead of `yield []`.

```suggestion
        yield [chr(0x72).chr(0x62), "72 62"];
        yield [chr(0x72).chr(0x62).chr(0x01).chr(0x00), "72 62 01 00"];
        yield [chr(0x72).chr(0x62).chr(0x01).chr(0x00).chr(0x00).chr(0x01), "72 62 01 00 00 01"];
        yield [chr(0x01).chr(0x00).chr(0x01), "01 00 01"];
```

Reply:
Good catch, adjusted as suggested :+1: